lib/simulator-xcode-6.js

```diff
@@ -571,6 +572,18 @@ class SimulatorXcode6 extends EventEmitter {
     }
   }
 
+  async enableCalendarAccess (bundleID) {
+    await (new Calendar(this.getDir(), bundleID)).enableCalendarAccess();
```

Comment:
this still feels a bit wonky to me. I think it's because we're creating Calendar objects every time, which nullifies any value in keeping state on an object. what if instead we did something like:

```js
this.calendar = new Calendar(this.getDir());
```

in the constructor. Then here we could do:

```js
await this.calendar.enableCalendarAccess(bundleID);
```

to me this makes more sense because bundleID isn't really an essential property of the Calendar class as you've defined it; I think it makes more sense to allow a sim object to have a cal object.

either that or we keep a hash of cal objects by bundle id on the sim object.